### PR TITLE
Mz config

### LINKED ISSUE #6: allow setting for env var GOOGLE_APPLICATION_CREDENTIALS
It is cumbersome to have to restart vscode with new env that points to another account file. We work with multiple customers and thus accounts.

So would you please create a setting to override the GOOGLE_APPLICATION_CREDENTIALS env var?

### PR DESCRIPTION
**Pull request by misak113**
closes #6

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to the "vscode-sops" extension will be documented in this fi
 
 Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
 
+## [Unreleased]
+### Added
+- Configuration `sops.enabled`, `sops.binPath` works now as expected.
+- Run control (config) file `.vscodesopsrc` for local project/workspace specific configuration (AWS profile & GCP credentials).
```
Comment (Morriz) [inline CHANGELOG.md:10]: Personally I would use something smaller like `.sopsrc` and claim it :) Let the sops devs also use it if they want!

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ All notable changes to the "vscode-sops" extension will be documented in this fi
...
+- Support for adjusting `sops.defaultAwsProfile` & `sops.defaultGcpCredentialsPath` configuration of vscode extension.
```
Comment (Morriz) [inline CHANGELOG.md:11]: no use prefixing these with 'default' I think, as they're a direct setting. Also
`awsProfile` and `gcpCredentialsPath` are shorter :)

Might be worth adding docs that say "empty > system defaults"

File: README.md [hunk 3]
```diff
@@ -19,8 +19,18 @@ VSCode extension which allows to realtime edit [SOPS](https://github.com/mozilla
 
 ## Extension Settings
 *Options are currently unavailable in 0.0.1 version*
-* `sops.enable`: enable/disable this extension
+* `sops.enable`: enable/disable this extension (default: true)
 * `sops.binPath`: Path to SOPS binary (default: executables from `$PATH`)
+* `sops.configPath`: Absolute path (Starts with /) or Relative path to project (Starts with ./) where the configuration for this extension is looking for (default: Looking for file `.vscodesopsrc` in root of project) See [Config file](#config-file) section.
+* `sops.defaultAwsProfile`: Default AWS profile name which will be used for sops command `--aws-profile` (default: uses from environment variable `$AWS_PROFILE`)
```
Comment (Morriz) [inline README.md:25]: Suggestion: remove "Default" and replace 
```
default: uses from environment variable `$AWS_PROFILE`
```
with
```
empty: defaults to environment variable `$AWS_PROFILE`
```

File: README.md [hunk 4]
```diff
@@ -19,8 +19,18 @@ VSCode extension which allows to realtime edit [SOPS](https://github.com/mozilla
 
 ## Extension Settings
 *Options are currently unavailable in 0.0.1 version*
-* `sops.enable`: enable/disable this extension
+* `sops.enable`: enable/disable this extension (default: true)
 * `sops.binPath`: Path to SOPS binary (default: executables from `$PATH`)
+* `sops.configPath`: Absolute path (Starts with /) or Relative path to project (Starts with ./) where the configuration for this extension is looking for (default: Looking for file `.vscodesopsrc` in root of project) See [Config file](#config-file) section.
```
Comment (Morriz) [inline README.md:24]: suggestion: replace
```
Absolute path (Starts with /) or Relative path to project (Starts with ./) where the configuration for this extension is looking for (default: Looking for file `.vscodesopsrc` in root of project) See [Config file](#config-file) section.
```
to
```
Path (absolute or relative) to the configuration for this extension (empty: defaults to `.vscodesopsrc` in root of project) See [Config file](#config-file) section.
```

**Review by Morriz — COMMENTED** [review]
Just gave some suggestions to docs and naming. Code looks fine but I  have no knowledge of it's intricacies.

Thanks a lot btw!

**Comment by Morriz** [comment]
I see it got merged already...I was too slow ;(

**Comment by misak113** [comment]
Hi, sorry for merging too soon. I've got a review internally by someone else. I'll go through your suggestions and merge fixes.

File: CHANGELOG.md (hunk 2 above)
Comment (misak113) [reply CHANGELOG.md:11]: I'll change that with prefixing "defaults." dot notation. From the clean code point of view, it's more obvious.

**Comment by misak113** [comment]
See changes here https://github.com/signageos/vscode-sops/pull/9

**Pull request merged**